loaders-app/src/spinners-data/SpinnerConfig.js

```diff
@@ -0,0 +1,3 @@
+const spinnerColor = '#1043A0';
```

Comment:
I found this which looks to allow you to access JSON config objects inside SASS files: https://www.viget.com/articles/sharing-data-between-sass-and-javascript-with-json/

Didn't look into it much though and it may be more trouble than its worth.  If not though, then we could store the color(s) configuration in a single js file as a JSON object.